scripts/simulate.js

```diff
@@ -134,19 +148,41 @@ async function fiatLeverage(amount) {
 }
 
 async function leverageCycle(amount) {
+  await network.provider.send("hardhat_setNextBlockBaseFeePerGas",
+    [`0x${Number(40*1e9).toString(16)}`]);
   const signer = (await ethers.getSigners())[0];
 
   const startingEth = await signer.getBalance();
 
   const ptBalance = await purchasePTs(amount);
   const fiatDebt = await collateralizeForFiat();
-  const daiBalance = await curveSwapFiatForDai();
+  const {daiBalance,effectiveFiatPrice,reserves0,reserves1, oldDaiBalance} = await curveSwapFiatForDai();
 
   const endingEth = await signer.getBalance();
   const gasDai = dsMath.wmul(startingEth.sub(endingEth), ETH_PRICE_DAI);
   const interestDai = dsMath.wmul(fiatDebt, ethers.utils.parseUnits((MATURITY_YEAR_FACTOR * FIAT_INTEREST_RATE * FIAT_PRICE_DAI).toFixed(DECIMALS).toString(), DECIMALS));
-  const daiBalanceOnMaturity = ptBalance.sub(gasDai).sub(interestDai).add(daiBalance).sub(dsMath.wmul(fiatDebt, ethers.utils.parseUnits(Number(FIAT_PRICE_DAI).toString())));
+  const slippage = daiBalance.sub(dsMath.wmul(fiatDebt, ethers.utils.parseUnits(Number(FIAT_PRICE_DAI).toString())))
```

Comment:
renamed to transaction fees (not currently used, I was thinking it would be clearer for logging, but left it disaggregated for now, since I'm not sure how best to track the cost of paying back the Fiat, and this way it's explicitly shown as a line item)